app/components/elements/breadcrumb_nav_component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Elements
+  # Displays the top bread crumb navigation
+  class BreadcrumbNavComponent < ApplicationComponent
+    renders_many :breadcrumbs, BreadcrumbComponent
+
+    def initialize(dashboard: true, admin: false)
+      @dashboard = dashboard
+      @admin = admin
+      super()
+    end
+
+    attr_reader :dashboard, :admin
```

Comment:
I'm not sure we've used this pattern consistently, but I have seen some spots where we do this instead:

```suggestion
    def dashboard?
      @dashboard
    end
    
    def admin?
      @admin
    end
```

Reply:
Thanks, made that change and it helps readability in the erb, I think.